api/reporting/curl/upload-snapshot-local.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Start hit the reporting snapshot & upload cron endpoint  locally.
+curl -X GET 'http://localhost:8081/v1/cron/uploadReportingSnapshot' \
+  --header "X-AppEngine-Cron: true" \
+  --header "Authorization: Bearer `gcloud auth print-access-token`" \
```

Comment:
nit: this header should be unnecessary - crons do not require user creds, else they wouldn't work when called by GAE